Events.php

```diff
@@ -26,19 +27,22 @@ public static function onAdminMenuInit($event)
         try {
             Yii::$app->moduleManager->getModule('custom_pages')->checkOldGlobalContent();
 
-            if (!Yii::$app->user->isAdmin()) {
+            $canManageModules = Yii::$app->user->can(new ManageModules());
```

Comment:
Hmm do we really need to split the permissions here? Users which can manage modules can simply disable modules or overwrite templates. I think its enough to say that users who can Manage modules also can do anything with custom pages, otherwise it gets rather complex. What do you think @luke- ? At least we need to document it somewhere otherwise its not obvious.

Reply:
@buddh4  I am not sure if I have understood you correctly.
In the previous version only Super Administrators could edit and manage custom pages.
With this PR, administrators with the `Manage Modules` permission can edit and manage custom pages (templates). (e.g. no need to access user data).

In addition, also regular users can edit custom pages with the `Manage Pages` permission.

Reply:
I think currently only users with `ManagePages` can edit/create pages and users with `ManageModules` can edit templates etc. Can you confirm this @yurabakhtin

Reply:
@buddh4 That's the case But I like the differentiation. Since templates or custom pages settings should not accessible for regular users with edit permission.

Reply:
Yes, thats ok, but the question is should users with `ManageModule` permission be allowed to edit pages which I think is not the case in this PR. Also we need to document this maybe in the module docs, otherwise its hard to see which permission is required to manage pages/templates.

Reply:
@buddh4

> I think in the current state of this PR only users with ManagePages can edit/create pages and users with ManageModules can edit templates etc. Can you confirm this @yurabakhtin

Yes, I confirm, In this PR we created a new permission `ManagePages`/"Can manage custom pages" to allow create/edit/delete custom pages. The core permissions `ManageModules` is used to allow users to change Settings/Config and Templates of the module "Custom Pages".

Reply:
@yurabakhtin I think a user only with the permission "Modul manager" should also be allowed to edit contents of the Custom Pages module. Or are there any arguments against it?

Reply:
@luke- I don't see any arguments against it. I think you better know what each permission can allow, so if you tell me to do this I will do, but yes it makes sense to allow `ManagePages` for all users who can `ManageModules` I think.

Reply:
@yurabakhtin Then do it like this, so everyone who has a `ManageModules` permission is allowed to manage the complete CustomPages module without restrictions. Someone with ManagePages can only manage pages.

Reply:
I agree the `ManageModules` permissions should be the main permission for managing modules and other modules can provide specialized permissions. Maybe even a `ManageTemplates` permission would be great so a group could be allowed to manage pages and templates. Otherwise a user who wants to manage templates needs the whole `ManageModules` permission.

Reply:
@luke- The fix to allow `ManagePages` for `ManageModules` in commit https://github.com/humhub/humhub-modules-custom-pages/pull/156/commits/a92500b5c8061ade011d32698a9eaea7251a48e0.

@buddh4 I don't find `ManageTemplates`. Do you mean to create this new one instead of `ManagePages`?

Reply:
I think he wanted to suggest to introduce another permission only for the Template Tab. But I think that is not necessary at the moment. I would suggest we wait here until the request for this comes from somebody.

@buddh4 Is that ok for you? I think we can then merge this PR?